superset-frontend/packages/superset-ui-core/src/query/getClientErrorObject.ts

```diff
@@ -255,7 +262,7 @@ export function getClientErrorObject(
     }
     resolve({
       ...responseObject,
-      error,
+      error: t(parseStringResponse(error)),
```

Comment:
from my understanding `t()` only works for hard-coded strings. I think behind the scene it's using static analysis while running `babel`. It's probably not a problem as is (no-op), but not useful either

Reply:
Just to note it, using `t()` on the string values inside the `ERROR_CODE_LOOKUP` object for some reason caused a failure in the `logging.test` file in `ui-core/test/utils` by causing `console` to evaluate to `undefined`. Maybe some kind of timeout? Whatever the cause I thought it was curious